fa_crosshair/init.sqf

```diff
@@ -10,11 +10,12 @@ dc_fac_fnc_crosshairInit = {
 		_crossHairImages = [
 			"\a3\ui_f\data\IGUI\Cfg\Cursors\weapon_ca.paa",
 			"\a3\ui_f\data\IGUI\Cfg\CrewAimIndicator\gunner_ca.paa",
+			"\a3\ui_f\data\IGUI\Cfg\WeaponCursors\coil_gs.paa",

```

Comment:
huh, we should probably use the actual weapon cursors instead of whatever I did, shouldn't we.